configs/components/leatherman.rb

```diff
@@ -10,7 +10,7 @@
   elsif platform.name =~ /solaris-10/
     pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/solaris/10/pl-gcc-4.8.2-1.#{platform.architecture}.pkg.gz"
     pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/solaris/10/pl-binutils-2.25.#{platform.architecture}.pkg.gz"
```

Comment:
I think there's a problem here. For solaris 10, we should be using pl-binutils 2.27, see:

https://github.com/puppetlabs/pl-build-tools-vanagon/blob/master/configs/components/binutils.rb#L3

The error that the bug report refers to mentions the error you were seeing when testing this PR. I bet this was the cause. I'm not sure though why 2.27 wasn't referenced in the components you removed pl-binutils from, though. Something's been broken here for a while.

Reply:
Good catch. Updated versions to match https://github.com/puppetlabs/pl-build-tools-vanagon/blob/fc4077e0a81fc1c168880b58e0d0ef2ad81f46b4/configs/components/boost.rb#L36-L37. Trying a rebuild now.

Reply:
Ok, that messed everything up. Going to have to do some digging later.